CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 The changelog for `IGListKit`. Also see the [releases](https://github.com/instagram/IGListKit/releases) on GitHub.
 
+## Master
+
+* Fixed `-[IGListAdapter reloadDataWithCompletion:]` not returning early when `collectionView` or `dataSource` is nil and `completion` is nil.  
+  [Ben Asher](https://github.com/benasher44)
+  [#51](https://github.com/Instagram/IGListKit/pull/51)
```

Comment:
I only left it as master, so that you all could change it when you all are ready to do the release. Happy to change it though if you all have a new version picked out already.